openghg/types/_enum.py

```diff
@@ -61,6 +61,12 @@ class ObsTypes(Enum):
     MOBILE = "ObsMobile"
 
 
+class OneD_types(Enum):
```

Comment:
```suggestion
class OneDTypes(Enum):
```

We should stick to CamelCase